engine/class_modules/paladin/sc_paladin_protection.cpp

```diff
@@ -113,6 +118,11 @@ struct avengers_shield_t : public avengers_shield_base_t
       p() -> cooldowns.avengers_shield -> reset( false );
       p() -> buffs.moment_of_glory -> decrement( 1 );
     }
+    if ( p() -> legendary.holy_avengers_engraved_sigil -> ok() && rng().roll( holy_avenger_proc_chance ) )
+    {
+      p() -> cooldowns.avengers_shield -> reset( false );
```

Comment:
The requires_reaction should be set to true here, since it's only a chance to proc.

Reply:
Yeah? Even though you have a whole gcd to react and know it's pretty likely?

Reply:
GCD duration is included in the reaction time. 
35% chance to proc is probably high enough to expect the occurence though, you're right.